integration/src/main/resources/configs/integration/DeleteWorkspaceWithPolicy.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "DeleteWorkspaceWithPolicy",
```

Comment:
As a heads up, this won't run as part of the PR test or nightly suites unless you add it to the FullIntegration suite (though that might be the right call until TPS is rolled out to environments > dev)